src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -1,10 +1,10 @@
 @(applicationId: ApplicationFormId, opportunity: Opportunity, sectionNumber: Int)
 
 @sectionLink(title: String, sectionNum: Int) = @{
-    if(sectionNum == sectionNumber) s"$sectionNum. $title"
+    if(sectionNum == sectionNumber) s"""<span class="part-number">$sectionNum.</span> $title"""
```

Comment:
I'd recommend using twirl syntax rather than dealing with string literals, i.e. = @{...} would be replaced by: 
= {
  <span class="part-number">@sectionNum.</span>
  @if(sectionNum == sectionNumber) {
      @title
  } else {
     <a href="@link">@title</a>
  }
}

see https://www.playframework.com/documentation/2.5.x/ScalaTemplates for more detail